cpp/test/matrix/linewise_op.cu

```diff
@@ -216,7 +216,6 @@ struct LinewiseTest : public ::testing::TestWithParam<typename ParamsReader::Par
     std::vector<std::tuple<I, I>> dims;
```

Comment:
While you're at this, would you mind making a benchmark out of the main test loop here ([run()](https://github.com/rapidsai/raft/blob/4718e5bc73b0235c34581ed7f6f9c320d7e7b347/cpp/test/matrix/linewise_op.cu#L151) function)?
I've made this test before we had benchmarks and used the NVTX annotations to benchmark kernel run times in various cases. Would be nice to have it in a proper place :)

Reply:
I've extended the matrix-vector op benchmark.